### ISSUE TITLE
[DetailsList][Accessibilty] Details list header column does not allow to pass header description (ariaDescription)

### ISSUE THREAD

**Issue by aadhars** [OfficeDev/office-ui-fabric-react#2645]
### Bug Report

- __Package version(s)__:  any Version
- __Browser and OS versions__: all Broswer

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes may be

Requested priority: Normla

Products/sites affected: Visual studio team services  VSTS

#### Describe the issue:

Details list does not allow user to set ariaDescription on column header. We have a scenario where we are allowing user to sort the column. Given the sort functionality we want to announce this functionality to user as the description of column header.

#### Actual behavior:

Details list does not allow to pass description for a column header.

#### Expected behavior:

There should be property exposed in IColumn interface of details list which accepts description of header and attach it to the column header button as its description.
